test/desdemona/ui/table_test.cljs

```diff
@@ -1,8 +1,43 @@
 (ns desdemona.ui.table-test
   (:require
    [desdemona.ui.table :as table]
-   [cljs.test :as t :refer-macros [deftest testing is are]]))
+   [cljs.test :as t :refer-macros [deftest testing is are]]
+   [reagent.core :as r]
+   [reagent.session :as session]
+   [cljs.core.match :refer-macros [match]]))
+
+(def isClient (not (nil? (try (.-document js/window)
+                              (catch js/Object e nil)))))
+
+(def rflush r/flush)
```

Comment:
Why does this get added here? We just didn't have client tests before today?

You might want to look at the equivalent code in shrieker because it cleans some of this up a little; e.g. `not nil?` is an antipattern &c